File: sharedlife-backend/src/main/java/net/tfg/sharedlife/dto/ConfirmedTaskDTO.java
package net.tfg.sharedlife.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import net.tfg.sharedlife.model.User;

import java.util.List;

@Getter
@Setter
@ToString
public class ConfirmedTaskDTO {
    private String username;
    private Long idTask;
    private List<UserDTO> members;
}

File: sharedlife-frontend/src/app/models/taskTraking/confirmedTaskDTO.ts
import { User } from "../user/user";

export class ConfirmedTaskDTO{
    username: string;
    idTask: string;
    members: User[];
    constructor(i?: string, u?: string, h?: User[]){
        this.idTask = i ?? "";
        this.username = u ?? "";
        this.members = h ?? [];
    }

}